editor/project_converter_3_to_4.cpp

```diff
@@ -278,6 +279,7 @@ static const char *gdscript_function_renames[][2] = {
 	{ "get_camera", "get_camera_3d" }, // Viewport -> this is also convertable to get_camera_2d, broke GLTFNode
 	{ "get_cancel", "get_cancel_button" }, // ConfirmationDialog
 	{ "get_caption", "_get_caption" }, // AnimationNode
+	{ "get_caret_blink_speed", "get_caret_blink_interval" }, // TextEdit, LineEdit
```

Comment:
This method name didn't exist in `3.x` so it doesn't need to be handled. Same for `set_caret_blink_speed`.